src/transformers/generation_logits_process.py

```diff
@@ -380,8 +380,9 @@ class NoBadWordsLogitsProcessor(LogitsProcessor):
 
     Args:
         bad_words_ids (`List[List[int]]`):
-            List of list of token ids that are not allowed to be generated. In order to get the tokens of the words
-            that should not appear in the generated text, use `tokenizer(bad_word, add_prefix_space=True).input_ids`.
+            List of list of token ids that are not allowed to be generated. In order to get the token ids of the words
+            that should not appear in the generated text, use `tokenizer(bad_words, add_prefix_space=True, 
```

Comment:
It should be a list of bad words, not a single bad word.
In addition, follow the fix in #15343